packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Added a new SDK method (`ConfigUtils.hasTokenExpired`) that allows developers to verify whether a given JSON web token has expired. [#2298](https://github.com/zowe/zowe-cli/pull/2298)
+- Enhancement: Added a new SDK method (`ProfileInfo.hasTokenExpiredForProfile`) that allows developers to verify whether a JSON web token has expired for a given profile. [#2298](https://github.com/zowe/zowe-cli/pull/2298)
```

Comment:
As the previous comment, better to turn this into an action: Use the new SDK method `ccc` to check whether a given JSON web token has expired for a given/specified profile.